ForumProject/investors/views.py

```diff
@@ -49,3 +72,9 @@ def create(self, request, *args, **kwargs):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    def destroy(self, request, *args, **kwargs):
+        if request.user.user_info.company_id != kwargs['pk']:
+            return Response({'error': 'You have no permission to delete investor'}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Returning a `400 Bad Request` for permission errors is not typical; `403 Forbidden` is more appropriate because it explicitly indicates that the operation is not allowed due to permissions

Reply:
Done.